config/socket_windows.go

```diff
@@ -0,0 +1,5 @@
+package config
+
+func setTransportAddress(cfg *Config) {
+	cfg.TransportAddress = `\\.\pipe\manifoldco\torus`
```

Comment:
Let's just add some documentation to the README as a caveat for windows support re: system wide. Then we can get this merged up.